tasklist/client/src/Tasks/CollapsiblePanel/CustomFiltersModal/DeleteFilterModal.tsx

```diff
@@ -30,8 +30,8 @@ const DeleteFilterModal: React.FC<Props> = ({
       danger
       open={isOpen}
       size="xs"
-      modalLabel="Delete filter"
-      modalHeading="The custom filter will be deleted."
+      modalLabel={isOpen ? 'Delete filter' : undefined}
+      modalHeading={isOpen ? 'The custom filter will be deleted.' : undefined}
```

Comment:
I'm doing this so it's easier to test, since Carbon never stops rendering invisible modals